Automattic/jetpack
# Add size control to sharing block

Fixes https://github.com/Automattic/jetpack/issues/35045

## Proposed changes:

Adding the possibility to change the size of the sharing button. Behavior is similar to the original Social Links block.

https://github.com/Automattic/jetpack/assets/60262784/86c7ef45-0126-4b8f-a869-5a3326bdb24d

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up your site with local changes
 - Make sure you have beta block enabled. In your environment (example Docker) wp-config.php you should have define( 'JETPACK_BLOCKS_VARIATION', 'beta' );
- Add Sharing Buttons block to a post.
- Add several buttons. On the main Sharing Buttons block you should see a Size setting.
